packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -570,6 +575,16 @@ class FlutterDriver {
     await _sendCommand(EnterText(text, timeout: timeout));
   }
 
+  /// TODO document and say never use this unless you really need it!
+  Future<void> setEditingState(String textEditingValue, { Duration timeout }) async {
+    await _sendCommand(SetEditingState(textEditingValue, timeout: timeout));
+  }
+
+  /// TODO document and say never use this unless you really need it!
+  Future<void> setMarkedText(String markedTextValue, { Duration timeout }) async {
```

Comment:
I added these three methods to `driver`.  `setMarkedText` might not be necessary, but I was using it to try to reproduce the crash (unsuccessfully) exactly as it happens.

Reply:
After looking into this with @HansMuller, the engine side of `setEditingState` seems to be unable to update the engine's native text input to contain marked text.